javascript/javascript3/week2/main.js

```diff
@@ -0,0 +1,56 @@
+const currencyFromSelect = document.getElementById('currency-from')
+const currencyToSelect = document.getElementById('currency-to')
+const currencyToGet = document.getElementById('currency-get')
+const amountInput = document.getElementById('amount')
+const resultDiv = document.getElementById('result')
+
+let exchangeRates
+let currencies
+
+// Make a request to the API to get exchange rates
+fetch('https://open.er-api.com/v6/latest/USD')
+  .then((response) => response.json())
+  .then((data) => {
+    exchangeRates = data.rates
+    currencies = Object.keys(exchangeRates)
+    // Populate the options for currencyFromSelect and currencyToSelect
+    currencies.forEach((currency) => {
+      const fromOption = document.createElement('option')
+      fromOption.value = currency
+      fromOption.textContent = currency
+      currencyFromSelect.appendChild(fromOption)
+
+      const toOption = document.createElement('option')
+      toOption.value = currency
+      toOption.textContent = currency
+      currencyToSelect.appendChild(toOption)
+    })
+  })
```

Comment:
Just for best practices it is a good idea to also add .catch(error => {}) construction to the end of Promise chain to handle errors of network request